master/buildbot/test/integration/test_upgrade.py

```diff
@@ -122,7 +122,13 @@ def assertModelMatches(self):
         def comp(engine):
             # use compare_model_to_db, which gets everything but indexes
             with engine.connect() as conn:
-                diff = compare_metadata(MigrationContext.configure(conn), self.db.model.metadata)
+                opts = None
+                if engine.dialect.name == 'mysql':
+                    # Disable type comparisons for mysql. Since 1.12.0 it is enabled by default.
+                    # https://alembic.sqlalchemy.org/en/latest/changelog.html#change-1.12.0
+                    # There is issue with comparison MEDIUMBLOB() vs LargeBinary(length=65536) in logchunks table.
+                    opts = {"compare_type": False}
```

Comment:
This fixes just test but is it correct for real migration?

Reply:
It's enough if the test is fixed for now. Do you still have logs from the error in test? Could you open a bug with these?

Reply:
>  Do you still have logs from the error in test? 

It is still visible in  PR #7447
See logs
- https://buildbot.buildbot.net/#/builders/89/builds/8025
- https://buildbot.buildbot.net/#/builders/115/builds/2474

Reply:
Thanks!